test/plugin_functional/test_suites/core_plugins/ui_settings.ts

```diff
@@ -48,7 +48,7 @@ export default function ({ getService, getPageObjects }: PluginFunctionalProvide
     });
 
     it('server plugins have access to registered settings', async () => {
-      await supertest.get('/api/ui-settings-plugin').expect(200).expect({ uiSettingsValue: 2 });
+      await supertest.get('/api/ui-settings-plugin').expect(200).expect({ uiSettingsValue: '2' });
```

Comment:
__note:__ Justification for the test change (the test seems to be broken in `main` and unintentional upgrade of`superagent` revealed that):

```http
GET /api/ui-settings-plugin HTTP/1.1
Host: localhost:5620
Accept-Encoding: gzip, deflate
User-Agent: node-superagent/3.8.3

HTTP/1.1 200 OK
content-type: application/json; charset=utf-8
accept-ranges: bytes

{"uiSettingsValue":"2"}
```